src/main/scala/org/broadinstitute/dsde/workbench/sam/util/LdapSupport.scala

```diff
@@ -36,27 +36,32 @@ trait LdapSupport extends DirectorySubjectNameSupport {
   protected def ldapSearchStream[T](baseDn: String, searchScope: SearchScope, filters: Filter*)(unmarshaller: Entry => T): Stream[T] =
     filters.flatMap { filter =>
       val search = new SearchRequest(baseDn, searchScope, filter)
-      val entrySource = new LDAPEntrySource(ldapConnectionPool.getConnection, search, true)
-      ldapEntrySourceStream(entrySource)(unmarshaller)
+      val connection = ldapConnectionPool.getConnection
+      val entrySource = new LDAPEntrySource(connection, search, false)
```

Comment:
notice that the last param switched from true to false

Reply:
nit: can you put the bool variable name in the call so it's clear what it does